### DISCUSSION TITLE
Osu is stuck at menu

### DISCUSSION THREAD

**Discussion by sTiKyt** [ppy/osu#14633] [Q&A] [answered]
I'm using the latest version from AUR, game is stuck after this animation:
![image](https://user-images.githubusercontent.com/35602029/132112470-c0e8198a-8ce3-4d12-8f90-5a6543abdef6.png)

It's staying on this screen:
![image](https://user-images.githubusercontent.com/35602029/132112480-df4898df-9757-469e-821a-c1e801036ed5.png)

It was working fine before, probably before one of the updates(but I don't remember, it's been like that for a while and I just ignored this game)
Logs from /.local/share/osu/logs:

[database.log](https://github.com/ppy/osu-framework/files/7110777/database.log)
[network.log](https://github.com/ppy/osu-framework/files/7110778/network.log)
[performance.log](https://github.com/ppy/osu-framework/files/7110779/performance.log)
[runtime.log](https://github.com/ppy/osu-framework/files/7110780/runtime.log)

I have no MIDI devices connected to my device, and JACK is disabled, I'm using pipewire-pulseaudio

Anyway, this is probably related to ppy/osu-framework#3297

Uupdating/reinstalling game doesn't help, and i'll repeat myself, it worked on this same system with no problems before

P.s. I updated game version after taking screenshots, it didn't change anything.

**Comment by peppy** [comment]
Some of your logs show last activity being March. Your database may be in a irrecoverable state. Try deleting it (`client.db`).

**Reply by sTiKyt** [reply]
Where is it located?

**Reply by sTiKyt** [reply]
ok, found it, my bad

**Reply by sTiKyt** [reply]
Unfortunately, still black after client.db removal
![image](https://user-images.githubusercontent.com/35602029/132112648-e78e5ed0-81da-4a7c-80d9-9298c0a886d5.png)
![image](https://user-images.githubusercontent.com/35602029/132112659-35d1cf16-d9f7-430e-9b0a-5c33ce31a6f4.png)

**Reply by peppy** [reply]
Do you have any custom rulesets loaded?

**Reply by sTiKyt** [reply]
I don't think so? I didn't set up anything custom myself, I used AUR package
https://aur.archlinux.org/packages/osu-lazer-bin/

Will update it now, to get the latest version and see if issue persists

**Reply by sTiKyt** [reply]
ok, new version number, old issue
![image](https://user-images.githubusercontent.com/35602029/132112728-8b7d83a0-5825-49db-9da4-94973cbd03ba.png)

**Reply by peppy** [reply]
Hmm, then you may be correct in thinking it's something to do with your audio setup. You might need to have a play with that at your end.

**Reply by sTiKyt** [reply]
But I don't have any idea what are the steps I can take to solve this myself

**Comment by PercyDan54** [comment]
You're on a outdated version(2021.524.0)

**Reply by sTiKyt** [reply]
Which I updated, check neighbor thread this is not related to issue I'm having

**Comment by gagahpangeran** [comment, marked as answer]
Try edit `~/.local/share/osu/game.ini` file and change `IntroSequence = Triangles` to `IntroSequence = Circles`, then launch the game again.

If it successfully open to main menu, then change your audio device in settings and see if it helps.

**Reply by sTiKyt** [reply]
Yep, and i was able to change back to Triangles successfuly! Thanks :D

**Reply by peppy** [reply]
how does that help though? are you playing without sound?

**Reply by bdach** [reply]
It helps because the circles intro isn't dependent on the beatmap track progressing (as the track is the backing clock in that case). See #7970, ppy/osu-framework#3297.

**Reply by peppy** [reply]
I understand that. I'm wondering how the user found this as a valid solution when if their sound was not working, getting past the intro should not change anything (their game would still not have sound).

**Reply by gagahpangeran** [reply]
They change their audio device via settings after successfully pass the intro I guess.

@stikyt is your sound working after changing the audio device via settings?

**Reply by peppy** [reply]
If that's the case, we definitely need more information. Because it sounds like the default device is at fault then, and may need upstream fixes.

**Reply by sTiKyt** [reply]
yes i changed device and sound works

**Reply by peppy** [reply]
from what to what?

**Reply by sTiKyt** [reply]
From "Default" to "PulseAudio Sound Server"

But "PipeWire Sounds Server" works fine as well, i'm using pipewire-pulse... No idea what does Osu!lazer consider to be default in my case, probably something non-existent

**Reply by peppy** [reply]
@sTiKyt we're made a potential fix for this which will be in the next release (see https://github.com/ppy/osu/pull/14708). If possible, could you please try reproducing the issue on the next release and see if you still get stuck on the black screen? We are unable to reproduce this so can't test whether it is improved at all.

**Reply by sTiKyt** [reply]
Of course I'll try, but let me know when it's released over this thread or something, that way I won't forget about it

**Reply by peppy** [reply]
@sTiKyt new release is live
